src/ui/Facet/FacetSettings.ts

```diff
@@ -157,7 +157,7 @@ export class FacetSettings extends FacetSort {
     new Popper(this.settingsButton, this.settingsPopup, {
       modifiers: {
         preventOverflow: {
-          boundariesElement: this.facet.root
+          boundariesElement: this.facet.element.parentElement
```

Comment:
How does that behave in "desktop"mode ?

Is this making it appear over the facet in desktop mode ?

Reply:
Yes like it's already the case see:
![image](https://user-images.githubusercontent.com/43446516/62391395-bae3e300-b532-11e9-8369-86bb5fbb1d87.png)

Reply:
![image](https://user-images.githubusercontent.com/43446516/62391566-304fb380-b533-11e9-8ef3-e4e2f49e9fcb.png)
Facet parent highlighted in blue

Reply:
Before : 
![image](https://user-images.githubusercontent.com/43446516/62391657-73118b80-b533-11e9-8282-7c7c584fc637.png)

Reply:
Alright, thanks for the screenshot ;)